docs/generate-docs.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+## Set up documentation so README.md is reflected onto apollographql.com/docs/android
```

Comment:
Can you add `set -e` at the beginning of this script to exit with an error if any of the commands failed ? (see https://intoli.com/blog/exit-on-errors-in-bash-scripts/).

This was the case before due to the `&&`. We should keep this behaviour.